src/features/import-projects/gitcoin/helpers.ts

```diff
@@ -6,16 +6,10 @@ import Showdown from "showdown";
 const generateGitcoinUrl = (project: GitcoinProjectInfo) => {
   const application = project.applications[0];
 
-  if (
-    !application ||
-    !application?.roundId ||
-    !application?.chainId ||
-    !application?.id
-  )
-    return "";
+  if (!application || !application?.id) return "";
 
-  const { chainId, roundId, id } = application;
-  return `#/round/${chainId}/${roundId}/${id}`;
+  const { id } = application;
+  return `#/projects/${id}`;
```

Comment:
It should be like this
```
return `#/projects/${project.id}`;
```
It's a sample graphql request for a single project
```json
 {
                "id": "0xffc984e7ef0daffd7a99d0b7b40e4cb18d7d1f9d99e0bce9e905e323e076ea43",
                "applications": [
                    {
                        "roundId": "25",
                        "chainId": 42161,
                        "id": "161"
                    }
                ]
            },
```

I suggest even removing `applications` section from GraphQL request